openshift/src/main/java/io/syndesis/openshift/OpenShiftServiceImpl.java

```diff
@@ -178,7 +200,7 @@ private void ensureBuildConfig(String name, DeploymentData deploymentData, Strin
               .withNewSourceStrategy()
                 .withNewFrom().withKind("ImageStreamTag").withName(builderStreamTag).endFrom()
                 .withIncremental(true)
-                .withEnv(new EnvVar("MAVEN_OPTS","-XX:+UseG1GC -XX:+UseStringDeduplication -Xmx500m", null))
+                .withEnv(new EnvVar("MAVEN_OPTS","-XX:+UseG1GC -XX:+UseStringDeduplication -Xmx300m", null))
```

Comment:
We might want to externalize this at some point...

Reply:
yeah, I even has comment there (but got lost ;) Happy for every PR